go/vt/dbconfigs/dbconfigs.go

```diff
@@ -156,6 +166,10 @@ func (dbcfgs *DBConfigs) IsZero() bool {
 	return dbcfgs.App.Uname == ""
 }
 
```

Comment:
just nit, i found this function's naming a bit confusing, and it might be better to do positive test asserting that `Uname` is provided, say `hasAppDebugUsername`